git-workflow/createFB.sh

```diff
@@ -112,38 +112,57 @@ ADDONS_DLP_TARGET_VERSION=1.1.x-$BRANCH-SNAPSHOT
 ADDONS_AGENDA_CONNECTORS_ORIGIN_VERSION=1.2.x-SNAPSHOT
 ADDONS_AGENDA_CONNECTORS_TARGET_VERSION=1.2.x-$BRANCH-SNAPSHOT
 
+function idrsaChmod() {
+	pushd ~/.ssh
+	chmod 600 id_rsa
+	popd
+}
+
```

Comment:
```suggestion
```
Unused and not needed IMO, Github actions checkout will manage this

Reply:
because an error occurred the first time for the permission, we can remove it or keep it dosen't effect anything, as you like.